.gitignore

```diff
@@ -12,6 +12,10 @@
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
+.idea/codeStyles
```

Comment:
These files should be kept

Reply:
But these are my local files, what is the reason they would be pushed into remote?